lib/core/task.toit

```diff
@@ -75,6 +76,90 @@ interface Task:
   static current -> Task:
     return Task_.current
 
+  /**
+  Divides the work given by $lambdas up into a number of
+    separate tasks that run concurrently.
+
+  Returns the results of running the $lambdas as a $Map
+    keyed by the lambda index. If the lambda at a given
+    index did not return, the results map will not contain
+    and entry for it. The results map is insertion ordered,
```

Comment:
I generally prefer not to use future tense, but it would work here.
Up to you.

```suggestion
    index doesn't return, the results map doesn't contain
    and entry for it. The results map is insertion ordered,
```